index.html

```diff
@@ -1,52 +1,352 @@
 </head>
+
 <body class="container">
     <header class="header-menu">
         <nav>
             <i href="#" class="fa fa-bars" id="hamburg-icon"></i>
         </nav>
-    </div> 
-    </header>    
-    <main> 
-        <div class="headline-container">
+        </div>
+    </header>
+
+    <main>
+        <section class="headline-section">
             <h1 class="headline-primary">Hey there. <br>
                 <span class="headline-primary">I'm Graciano.</span> <br>
-                <span class="headline-secondary">I'm a software developer</span>     
+                <span class="headline-secondary">I'm a software developer</span>
             </h1>
-         </div>
-         <div class="headline-content">
-            <p>
-                I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded,         
-                  don't hesitate to contact me.
-            </p>
-         </div> 
-
-         <ul class="socialmedia-links">
-            <li><a href="https://twitter.com/GracianoSoft"><img src="assets/img/twitter.svg" alt=""></a></li>
-            <li><a href="https://www.linkedin.com/in/graciano-manuel-henrique-175527221/"><img src="assets/img/Linkedin icon.svg" alt=""></a></li>
-            <li><a href="https://medium.com/@gracianomanuelhenrique"><img src="assets/img/medium.svg" alt=""></a></li>
-            <li><a href=" https://github.com/Graciano1997"><img src="assets/img/github.svg" alt=""></a></li>
-            <li><a href="#"><img src="assets/img/angellist.svg" alt=""></a></li>
-            
-         </ul> 
-        
-     
-        
+            <div class="headline-content">
+                <p>
+                    I can help you build a product , feature or website Look through some of my work and experience! If
+                    you like what you see and have a project you need coded,
+                    don't hesitate to contact me.
+                </p>
+            </div>
+
+            <ul class="socialmedia-links">
+                <li><a href="https://twitter.com/GracianoSoft"><img src="assets/img/twitter.svg" alt=""></a></li>
+                <li><a href="https://www.linkedin.com/in/graciano-manuel-henrique-175527221/"><img
+                            src="assets/img/Linkedin icon.svg" alt=""></a></li>
+                <li><a href="https://medium.com/@gracianomanuelhenrique"><img src="assets/img/medium.svg" alt=""></a>
+                </li>
+                <li><a href=" https://github.com/Graciano1997"><img src="assets/img/github.svg" alt=""></a></li>
+                <li><a href="#"><img src="assets/img/angellist.svg" alt=""></a></li>
+            </ul>
+            <div class="btn-hover"><img src="assets/img/see.svg" alt="" srcset=""></div>
+        </section>
+
+        <section class="works-section">
+            <div class="headline-container">
+                <h1 class="headline-primary">My Recent <br>
```

Comment:
- [ ] kindly consider giving this header another hx like h2 because using only one H1 creates hierarchy and gives your page a more logical structure